website/docs/r/nsxt_distributed_firewall.html.markdown

```diff
@@ -0,0 +1,129 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_distributed_firewall"
+sidebar_current: "docs-vcd-resource-nsxt-distributed-firewall"
+description: |-
+  The Distributed Firewall allows user to segment organization virtual data center entities, such as
+  virtual machines, based on virtual machine names and attributes. 
+---
+
+# vcd\_nsxt\_distributed\_firewall
+
+The Distributed Firewall allows user to segment organization virtual data center entities, such as
+virtual machines, based on virtual machine names and attributes. 
+
+## Example Usage
+
+```hcl
+
+data "vcd_nsxt_network_context_profile" "cp1" {
+  context_id = vcd_vdc_group.test1.id
+  name       = "CTRXICA"
+  scope      = "SYSTEM"
+}
+
+resource "vcd_nsxt_distributed_firewall" "t1" {
+  org          = "datacloud"
+  vdc_group_id = vcd_vdc_group.test1.id
```

Comment:
Maybe it's worth it to add vdc group datasource

Reply:
Added